src/ORM/FieldType/DBInt.php

```diff
@@ -64,7 +64,13 @@ public function Nice()
     {
         return sprintf('%d', $this->value);
     }
-
+    
+    public function SpelledOut()
+    {
+        $v = $this->prepValueForDB($this->value);
+        return (new NumberFormatter(i18n::get_locale(), NumberFormatter::SPELLOUT))->format($v);
```

Comment:
```suggestion
        $value = $this->prepValueForDB($this->value);
        $formatter = new NumberFormatter(i18n::get_locale(), NumberFormatter::SPELLOUT);
        return $formatter->format($value);
```
Please don't use single-letter variables, nor wrap instantiation in quotes.